hangman/game.py

```diff
@@ -1,13 +1,131 @@
 from .exceptions import *
-
+import random
 
 class GuessAttempt(object):
-    pass
-
+    
+    def __init__(self, character, hit=False, miss=False):
+        self.character = character
+        self.miss = miss
+        self.hit = hit
+        
+        if hit and miss:
+            raise InvalidGuessAttempt
+            
+        
+    def is_hit(self):
+        if self.hit:
+            return True
+        
+        if not self.hit: 
+            return False
+                
+        
+    def is_miss(self):
+        if self.miss: 
+            return True
+        
+        if not self.miss: 
+            return False
+            
 
 class GuessWord(object):
-    pass
+    def __init__(self, word_to_guess):
+         
+        self.answer = word_to_guess.lower()  
+        
+        if not word_to_guess:
+            raise InvalidWordException
+        else:
+            self.masked = len(word_to_guess)*"*"
+        
```

Comment:
we can consider shifting all exceptions at the top and initialization of variables after that for readability.. so something like :
```python
if not word_to_guess:
   raise InvalidWordException

self.answer = word_to_guess.lower() 
self.masked = len(word_to_guess)*"*"
```